network/codec/cbor/msg_channel_mapping.go

```diff
@@ -55,4 +55,8 @@ func initializeChannelToMsgCodesMap() {
 
 	// dkg
 	ChannelToMsgCodes[channels.DKGCommittee] = []uint8{CodeDKGMessage}
+
+	// cluster based channels
+	ChannelToMsgCodes[channels.SyncClusterPrefix] = []uint8{CodeSyncRequest, CodeSyncResponse, CodeRangeRequest, CodeBatchRequest, CodeBlockResponse}
```

Comment:
Is it fine to have `CodeRangeRequest` for more than one channel? same applies to other message codes as well. If it is should not be the case, please fix it.

Reply:
It is okay because sync-committee and sync-cluster use the same message codes unlike the consensus-cluster which has it's own CodeClusterBlockVote etc message codes. I am not sure why we don't also have CodeClusterSyncRequest etc for sync-cluster.